Add back warning to not remove inference id in semantic_text for 8.15 (#111626)
elastic/elasticsearch: docs/reference/mapping/types/semantic-text.asciidoc

The `inference_id` will not be validated when the mapping is created, but when documents are ingested into the index.
When the first document is indexed, the `inference_id` will be used to generate underlying indexing structures for the field.

> WARNING: Removing an {infer}  endpoint will cause ingestion of documents and semantic queries to fail on indices that define `semantic_text` fields with that {infer}  endpoint as their `inference_id`.

Reviewer: ```suggestion
WARNING: Removing an {infer}  endpoint will cause ingestion of documents and semantic queries to fail on indices that define `semantic_text` fields with that {infer} endpoint as their `inference_id`.
```

> Please check that {infer}  endpoints are not used in `semantic_text` fields before removal.

Reviewer: ```suggestion
Before removal, check if {infer} endpoints are used in `semantic_text` fields.
```
